### DISCUSSION TITLE
Landing Detector VTOL support (multicopter phase)

### DISCUSSION THREAD

**Discussion by Jetrell** [iNavFlight/inav#9323] [Ideas]
@breadoven I thought I'd raise this issue with you here.. Shota tried to ping you a couple of weeks back on Discord concerning the matter.
The Landing detector issue appears on conventional VTOL air frames, in the multicopter landing phase.
This issue is seen as the VTOL tries to make a decent when a strong wind is present.
The wind blowing over the aerofoil will create enough lift to either cause the VTOL to hold altitude, or slow its decent considerably.. Which in-turn causes the landing detector to falsely disarm the VTOL in the landing phase... Which is understandable, due to the aircraft not falling at a velocity that would be expected for a multicopter. Which can not float if the throttle is lowered.

Shota was hesitant to make any modifications to the landing detector code. And has added a recommendation to disable `nav_disarm_on_landing = OFF` for VTOL support.
Since you are the architect of the Landing detector. I thought I'd raise the matter in case you weren't aware of it.. And maybe one day have the time to look at an alternative logic for this type of support.

**Comment by breadoven** [comment]
I assume this is caused by the throttle being low so the plane is effectively diving into wind (to avoid being blown back) with the motors off but with little horizontal or vertical airspeed ? Is the disarm occurring in a sim or from actual flight testing ? I haven't looked at the VTOL stuff so it's not clear whether this is using multirotor or fixed wing landing detection when this happens, I'm guessing multirotor ?

**Comment by Jetrell** [comment]
Yes, it was as you have assumed.. And using the multirotor landing detector.
Here is the flight.
https://youtu.be/ACgL36P4DDg?si=BjWqShcvnDIFS4k-&t=222

**Comment by breadoven** [comment]
It's not to do with the throttle being low it's just that it's in the RTH Landing phase which means the landing detector is looking for a landing and I think It'll be using the long standing landing detection method used for RTH in that case, based on average throttle during descent + velocities. It'll need some additional checks probably to deal with this VTOL situation to get around this problem. It's possible just checking for gyro movement might be a fix given how much it appeared to be moving around in the video.
